v0/pkg/prometheus/gen/labels.go

```diff
@@ -28,10 +38,7 @@ func MakeLabelReflector(t reflect.Type, parent string, parentLabelNames LabelNam
 	}
 
 	var generators []label.KeyValueGenerator
-	labelNames := LabelNames{}
-	for _, ln := range parentLabelNames {
-		labelNames = append(labelNames, ln)
-	}
+	labelNames := append(LabelNames{}, parentLabelNames...)
```

Comment:
There should hopefully be a more elegant way of doing this?

Reply:
Could not find a more elegant way :(